cmd/validator/validator.go

```diff
@@ -5,222 +5,200 @@ validates them using the go package for each configuration type.
 Currently Apple PList XML, CSV, HCL, HOCON, INI, JSON, Properties, TOML, XML, and YAML.
 configuration file types are supported.
 
-Usage: validator [OPTIONS] [<search_path>...]
-
-positional arguments:
-    search_path: The search path on the filesystem for configuration files. Defaults to the current working directory if no search_path provided. Multiple search paths can be declared separated by a space.
-
-optional flags:
-  -depth int
-    	Depth of recursion for the provided search paths. Set depth to 0 to disable recursive path traversal
-  -exclude-dirs string
-    	Subdirectories to exclude when searching for configuration files
-  -exclude-file-types string
-    	A comma separated list of file types to ignore
-  -output
-     	Destination of a file to outputting results
-  -reporter string
-    	Format of the printed report. Options are standard and json (default "standard")
-  -version
-    	Version prints the release version of validator
+Cross Platform tool to validate configuration files
+
+Usage:
+  validator [flags]
+  validator [command]
```

Comment:
You think we should maintain this comment here? Since running -h already outputs the usage?